webknossos/examples/image_stack_to_dataset.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from webknossos import Dataset
+from webknossos.dataset import COLOR_CATEGORY
+
+INPUT_FILES = (
+    Path(__file__).parent.parent / "testdata" / "tiff" / "test.0000.tiff",
+    Path(__file__).parent.parent / "testdata" / "tiff" / "test.0001.tiff",
+)
+OUTPUT_FOLDER = Path("testoutput/tiff_dataset")
+
+
+def main() -> None:
+    """Convert a list of images into a WEBKNOSSOS dataset and directly add them as a new layer."""
+    dataset = Dataset(
+        dataset_path=OUTPUT_FOLDER,
+        voxel_size=(11, 11, 11),
+        name="My_new_dataset",
+        exist_ok=False,
+    )
+    dataset.add_layer_from_images(
+        images=INPUT_FILES,
+        layer_name="test",
```

Comment:
Please use a more fitting name. I assume the test tiffs are from the cremi dataset, then perhaps "raw EM" or "grayscale EM" or "raw CREMI" or something is probably more fitting.